app/websocket/app/main/model_manager.py

```diff
@@ -0,0 +1,188 @@
+import syft as sy
+
+import pickle
+import os
+
+from .persistence.models import db, SyftModel
+from sqlalchemy.exc import SQLAlchemyError, IntegrityError
+
+
+model_cache = dict()
+
+
+def cache_get_model(model_id: str):
+    """Checks the cache for a model. If model not found, returns None.
+    
+    Args:
+        model_id (str): Unique id representing the model. 
+
+    Returns:
+        A model encoded in ("ISO-8859-1"), else returns None.
+
+    """
+    model_ids = model_cache.keys()
+    if model_id in model_ids:
+        # model found
+        return model_cache[model_id]
+
+    return None
+
+
+def cache_save_model(serialized_model: bytes, model_id: str):
+    """Saves the model to cache. Will fail if a model with same id already exists.
+
+    Args:
+        serialized_model (bytes): The model object to be saved. encoded in ("ISO-8859-1").
+        model_id (str): The unique identifier associated with the model.
+
+    Returns:
+        True for success, False otherwise.
+
+    """
+    if cache_get_model(model_id) is None:
+        model_cache[model_id] = serialized_model
+        return True
+    else:
+        return False
+
+
+def cache_del_model(model_id: str):
+    """Deletes the given model_id. If it fails (model not present) it returns False.
+    
+    Args:
+        model_id (str): Unique id representing the model. 
+
+    Returns:
+        True is deleted, false otherwise.
+
+    """
+    if cache_get_model(model_id) is None:
+        return False
+    else:
+        del model_cache[model_id]
+
+
+def models_list():
+    """Returns a list of currently existing models. Will always fetch from db.
+
+    Returns:
+        A list object, containig model_id of all the models.
+
+    """
+    try:
+        result = db.session.query(SyftModel.id).all()
+        model_names = []
+        for model in result:
+            model_names.append(model.id)
+        return model_names
+    except SQLAlchemyError as e:
+        # probably no model found with the model_id specified
+        return None
+
+
+def save_model_for_serving(serialized_model: bytes, model_id: str):
+    """Saves the model for later usage. 
+
+    Args:
+        serialized_model (bytes): The model object to be saved. encoded in ("ISO-8859-1").
+        model_id (str): The unique identifier associated with the model.
+
+    Returns:
+        True for success, False otherwise.
```

Comment:
You could return an actual response (dict) from this function.

Reply:
A dictionary storing a message that is returned from the API. If the models is stored successfully return {"success": "Model deployed..."} else return {"error": "exception message"}